packages/synapse-react-client/src/lib/containers/CopyToClipboardInput.tsx

```diff
@@ -46,25 +46,32 @@ export const CopyToClipboardInput: React.FunctionComponent<
         autohide={true}
       ></ToastMessage>
       <div
-        className="SRC-copyToClipboardInputContainer SRC-marginBottomTop"
+        style={{
+          margin: '15px 0px',
+          display: 'flex',
+          justifyContent: 'center',
+        }}
```

Comment:
Could we instead use MUI `Box` and then this instead?

```
display="flex"
justifyContent="center"
sx={{
	my: 2,
	mx: 0, 
}}
```

Nbd if it doesn't play nice with the ref or something

Reply:
Works well! Will be included in the next commit